### ISSUE TITLE
Modal: Close Icon optimization

### ISSUE THREAD

**Issue by orangiene** [humhub/humhub#5506]
### Is your feature request related to a problem? Please describe.
The light gray Close icon is barely visible, should be optimized.

### Describe the solution you'd like
Enlarge the icon and use the primary color for more accessibility.
Close Icon in 16x16px

![modal-close-icon](https://user-images.githubusercontent.com/62644500/150413853-9e0b732c-6013-4399-ba2a-5a717fa27557.jpg)
